runtime/near-vm-runner/Cargo.toml

```diff
@@ -19,12 +19,11 @@ bn.workspace = true
 borsh.workspace = true
 ed25519-dalek.workspace = true
 enum-map.workspace = true
-finite-wasm = { workspace = true, features = ["instrument"] }
-loupe.workspace = true
```

Comment:
Note for myself: we should probably introduce cargo-udeps in our CI, to avoid unused crates living in our Cargo.toml’s

Reply:
Definitely :)